### ISSUE TITLE
Storing accuracy and losses

### ISSUE THREAD

**Issue by atakann** [divamgupta/image-segmentation-keras#62]
Hi,

Thank you for your implementation it helped me a lot to understand the concepts.

Can I store the accuracy and loss values and then use them for plotting? I could not figure out using the metrics part of the training. Currently I am just manually copying the loss and accuracy values to plot the results.

And also, I read from one of the posts that pixel-level (TP+TN)/Total was used. Is there any way to change these, play with these loss outcomes?

Thank you in advance.

**Comment by divamgupta** [comment]
https://medium.com/@kapilvarshney/how-to-plot-the-model-training-in-keras-using-custom-callback-function-and-using-tensorboard-41e4ce3cb401 here you go.

You can define your own metrics and attach them to the model while calling compile. : https://stackoverflow.com/questions/37657260/how-to-implement-custom-metric-in-keras

**Comment by mano-96** [comment]
Hi, did you manage to plot the graphs?
If yes, could you help me?
Thank you in advance.
